util/htmlTemplate.ts

```diff
@@ -20,22 +19,22 @@ export type TemplateOpts = {
 };
 
 const resolveRelativePath = (from: string, to: string) => {
-  const raw_path = path.posix.join(
+  const rawPath = path.posix.join(
     path.relative(path.dirname(from), path.dirname(to)),
     path.basename(to),
   );
 
-  return raw_path.startsWith(".") ? raw_path : `./${raw_path}`;
+  return rawPath.startsWith(".") ? rawPath : `./${rawPath}`;
 };
 
-export const htmlTemplate = async ({
+export const htmlTemplate = ({
   attributes,
   files,
   meta,
   path,
   title,
   bodyEntry,
-}: TemplateOpts): Promise<string> => {
+}: TemplateOpts): string => {
```

Comment:
I think it's better to keep the return type of this function because this is used in the recipe and is considered as a part of public API. So changing this would break the user's code.